content-src/components/DiscoveryStreamComponents/Navigation/_Navigation.scss

```diff
@@ -0,0 +1,44 @@
+.ds-navigation {
+
+  &.ds-navigation-centered {
+    text-align: center;
+  }
+
+  &.ds-navigation-right-aligned {
+    text-align: right;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  ul li {
+    display: inline-block;
+
+    &::after {
+      content: '•';
+      padding: 8px;
+    }
+
+    &:last-child::after {
+      content: none;
+    }
+
+    a {
+      color: $blue-60;
+    }
+  }
+
+  .ds-more-recommendations::after {
+    background: url('../data/content/assets/topic-show-more-12.svg') no-repeat center center;
```

Comment:
The designs (at least for the 2/3 and 1/3 columns) don't show the arrow. Is this intended to be split out from this PR?

Reply:
Yeah I didn't notice that.

To me it actually looks like the more recommendation link isn't displayed in the 2/3 and 1/3 views at all, which the arrow is implicitly tied to and only displayed in the last item. So no more recs link, no more arrows. The more recs link is also not explicitly defined, it's just the last item in the array that the server returns. I treat all of these as links the same and I don't have a real sense as to which is which. So I think the best I can do is always hide the last item in the 1/3 and 2/3 cases, and get assume that the server always has a more recs link as the last item. If for some reason the server stops sending back a more recs link as the last item, we'll start unintentionally hiding the last item in 1/2 and 2/3.

Reply:
Other options are, the server explicitly returns a prop for more recs link instead of having it in the end of the array, or we somehow return the array in the string.

Reply:
I think for now I'll just hide the last item in the 1/3 and 2/3 while we discus this with the server. Pending there reaction, we may or may not open a new ticket to make this more explicit and less prone to error.

Reply:
I'm actually seeing in The example comps there are 1/3 and 2/3 navigation elements with 1 item and an arrow, so I think we shouldn't do anything here.